lib/govspeak.rb

```diff
@@ -29,6 +29,7 @@ class Document
     Parser = Kramdown::Parser::KramdownWithAutomaticExternalLinks
     PARSER_CLASS_NAME = Parser.name.split("::").last
     UUID_REGEX = /^[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$/i.freeze
+    NEW_PARAGRAPH_LOOKBEHIND = %q{(?<=\A|\n\n|\r\n\r\n)}.freeze
```

Comment:
I noticed in the commit message that you mentioned a 'lookahead' but it's 'lookbehind' in the code so wanted to check what the correct term is?

Reply:
d'oh... thanks. It is indeed a lookbehind.